### ISSUE TITLE
Roomba2 not registered by any Plugin

### ISSUE THREAD

**Issue by clipse2004** [iRayanKhan/homebridge-roomba2#1]
HI Dude, 

after installation i always get this error, before this plugin i used roomba-stv and everything works fine. 

### Steps to Reproduce (for bugs)
Error: The requested accessory 'Roomba2' was not registered by any plugin.
    at PluginManager.getPluginForAccessory (/usr/local/lib/node_modules/homebridge/src/pluginManager.ts:198:15)
    at /usr/local/lib/node_modules/homebridge/src/server.ts:345:37
    at Array.forEach (<anonymous>)
    at Server._loadAccessories (/usr/local/lib/node_modules/homebridge/src/server.ts:328:29)
    at Server.start (/usr/local/lib/node_modules/homebridge/src/server.ts:154:12)
    at cli (/usr/local/lib/node_modules/homebridge/src/cli.ts:80:10)
    at Object.<anonymous> (/usr/local/lib/node_modules/homebridge/bin/homebridge:17:22)
    at Module._compile (internal/modules/cjs/loader.js:1063:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)
    at Module.load (internal/modules/cjs/loader.js:928:32)
    at Function.Module._load (internal/modules/cjs/loader.js:769:14)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:72:12)
    at internal/main/run_main_module.js:17:47

### Versions
- **Homebridge**: `homebridge --version`newest
- **Node**:       `node 14`
- **Plugin**:         0.01

### Configuration
        {
            "name": "Roomba",
            "model": "s9",
            "blid": "XXXXX",
            "robotpwd": "xxxxxxxxxxx",
            "ipaddress": "192.168.178.xx",
            "autoRefreshEnabled": true,
            "dockContactSensor": true,
            "runningContactSensor": true,
            "cacheTTL": 10,
            "accessory": "Roomba2"
        }

```

### Homebridge Log / Command Output
<!-- Paste relevant output between the two ``` lines below
  - Remove any sensitive information, passwords, etc.
  - If output is long:
    - Please extract only the relevant portions if possible
    - Consider creating a gist, etc
  - However, please include the beginning of the log where the homebridge initialization happens if possible
-->
```

```

**Comment by sharkannon** [comment]
Looks like a bug in the default config.. if you edit the accessory to read "Roomba" vs "Roomba2" it appears to work.  Not sure if that's intentional or not, but for sure a bug :(

**Comment by iRayanKhan** [comment]
Yup, it's a bug. Thanks for reporting it, it'll be fixed before V1.0

**Comment by clipse2004** [comment]
You mean I should edit config in Homebridge to Roomba only?

**Comment by sharkannon** [comment]
That's what I did.. what "iRayanKhan"'s fix is I don't know

`   {
        "name": "Roomba",
        "model": "s9",
        "blid": "XXXXX",
        "robotpwd": "xxxxxxxxxxx",
        "ipaddress": "192.168.178.xx",
        "autoRefreshEnabled": true,
        "dockContactSensor": true,
        "runningContactSensor": true,
        "cacheTTL": 10,
        "accessory": "Roomba"
    }`

Should hopefully fix it, at least it got rid of the error on mine anyway :)

**Comment by iRayanKhan** [comment]
Yup, changing the last line to Roomba instead of Roomba2 should work. Ill have it updated to make it use Roomba2 to avoid any confusion.

**Comment by clipse2004** [comment]
If I change this, the plugin will run, but it will be constantly updated and my HomeKit app will be extremely slow. 
I'll wait until you have adjusted this ;)

**Comment by iRayanKhan** [comment]
This has been fixed in v0.0.2, V1.0.0 is out now.

**Comment by iRayanKhan** [comment]
Note:

You may get an error if you try to update the plugin using the webUI or (config-ui-x).

To fix this, manually uninstall the plugin:

```
sudo npm remove -g homebridge-roomba2
```

Then, re-install the plugin:
```
sudo npm i -g homebridge-roomba2
```

Make sure to update Roomba, to Roomba2.
